package.json

```diff
@@ -2,15 +2,19 @@
   "name": "outline-client",
   "productName": "Outline",
   "scripts": {
-    "action": "node ./src/build/run_action.mjs",
     "action:help": "npm run action list",
     "action:list": "npm run action list",
+    "action": "node ./src/build/run_action.mjs",
     "clean": "rm -rf build node_modules www platforms plugins",
-    "format": "pretty-quick --staged --pattern \"**/*.{cjs,mjs,html,js,json,md,ts}\"",
     "format:all": "prettier --write \"**/*.{cjs,mjs,html,js,json,md,ts}\"",
-    "lint": "npm run lint:ts",
+    "format": "pretty-quick --staged --pattern \"**/*.{cjs,mjs,html,js,json,md,ts}\"",
     "lint:ts": "eslint 'src/**/*.ts'",
-    "reset": "npm run clean && npm ci"
+    "lint": "npm run lint:ts",
+    "reset": "npm run clean && npm ci",
+    "start:electron": "OUTLINE_DEBUG=true electron .",
+    "start:www": "webpack serve --mode=development --static=platforms/browser/www --config=src/www/webpack_cordova.mjs --open /index_cordova.html",
```

Comment:
We need to figure out how this works on Windows.

Reply:
also figure out a naming convention we like

Reply:
maybe we need https://www.npmjs.com/package/cross-env?

Reply:
Seems cross-env is a common solution. `OUTLINE_DEBUG=true` definitely will not work on Windows.

Reply:
Even with cross-env, wouldn't we still have issues with the paths?

Reply:
It seems we would: https://stackoverflow.com/questions/50998089/running-npm-script-on-windows-starting-with-a-period

Reply:
I think I just bite the bullet and write these one-liners in JS